### ISSUE TITLE
HighchartsReactNative Not working if expo published 

### ISSUE THREAD

**Issue by maxim-malyavko** [highcharts/highcharts-react-native#70]
Hello team! HighchartsReactNative still does not work after publish. I use expo. publish: ios.
I saw that similar questions were asked a month ago, but they were closed, and there was no solution.
What's happening? Locally, expo draws graphics; after loading, a white square is displayed in the appstore.
node -v v12.16.2
react-native-cli: 2.0.1
react-native: 0.59.8
expo-cli 3.17.5
expo 35.0.0
sdk 35
react-native-highcharts  ^1.0.2

with react-native-highcharts  2.0.0 not show same...(((
If there are no solutions, tell me about the process. Thank.

**Comment by sebastianbochan** [comment]
Do you receive any errors or only blank screen?
Have you tried to copy files into right directions?

**Comment by MaximMalyavko85** [comment]
I almost refused to use this library and was ready to rewrite the code using another one. It was very angry, especially since you see the result (more precisely, a blank screen) only after unloading in the appstore. But the project has already started and many customers are already using it. I have not received a response from you, and 5 days for your reaction (answer) - connivance on your part. I delved into the Internet, but many answers related to early SDKs. I tried to set the flag useSSL = {true} useCDN = {true}, and, oh, magic - the graphs began to be reflected. I was doubly angry, because it was just a finger in the sky and it was necessary to test only after unloading and validating Apple. Also, I did not fully realize what these 2 attributes do. You can give a detailed answer, why they exist and what problems they solve.
Thank.

**Comment by sebastianbochan** [comment]
Hi @MaximMalyavko85,
Im really sorry that you were annoyed and disappointed. All parameters are described in the README, I hope that the docs are helpful.

https://github.com/highcharts/highcharts-react-native#useSSL

**Comment by sebastianbochan** [comment]
We just finished testing the wrapper and released the beta version which is available here https://github.com/highcharts/highcharts-react-native/tree/beta-3

Any feedback will be really helpful.

Please keep in mind, that all packages should be in the latest versions of them. Number of versions are declared also in the `package.json` file.

**Comment by sebastianbochan** [comment]
The new 3.0 version is released. 

Im closing this ticket. If you will encounter any problems, please create a new ticket. In this case we will avoid confusing and keep the newest version on track.
